modules/nf-core/jvarkit/vcfpolyx/main.nf

```diff
@@ -0,0 +1,71 @@
+/**
+ * JVARKIT_VCFPOLYX
+ * Author: Pierre Lindenbaum PhD
+ * vcfpolyx is a sub-command of the jvarkit package. It is used to annotate a vcf for the poly-x repeats
+ */
+process JVARKIT_VCFPOLYX {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/jvarkit:2024.08.25--hdfd78af_1':
+        'biocontainers/jvarkit:2024.08.25--hdfd78af_1' }"
+
+    input:
+    tuple val(meta),  path(vcf)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+    tuple val(meta4), path(dict)
+
+    output:
+    tuple val(meta), path("*.${extension}"), emit: vcf
+    tuple val(meta), path("*.tbi")         , emit: tbi, optional: true
+    tuple val(meta), path("*.csi")         , emit: csi, optional: true
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args1  = task.ext.args1 ?: ''
+    def args2  = meta.vcfpolyx_args ?: (task.ext.args2 ?: ' --tag POLYX --max-repeats 10 ')
+    def args3  = task.ext.args3 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    extension =     args3.contains("--output-type b") || args3.contains("-Ob") ? "bcf.gz" :
+                    args3.contains("--output-type u") || args3.contains("-Ou") ? "bcf" :
+                    args3.contains("--output-type z") || args3.contains("-Oz") ? "vcf.gz" :
+                    args3.contains("--output-type v") || args3.contains("-Ov") ? "vcf" :
+                    "vcf"
+
+    if ("$vcf" == "${prefix}.${extension}") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+    """
+    set -o pipefail
```

Comment:
```suggestion
```

I don't think we need that here as it should be handled by Nextflow already

Reply:
> it should be handled by Nextflow already

are you sure about this ? e.g: https://github.com/nextflow-io/nextflow/discussions/3792

Reply:
this is in the TEMPLATE: https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L224

Reply:
so yeah, Nextflow will handle it for you if it's specified in your config, which is for nf-core pipelines

Reply:
@maxulysse oh ! ok, good to know !